Various changes to documentation styling (#238)
zen-mod/ZEN: docs/development/coding_guidelines.md

### 1.1 Variable Names

#### 1.1.1 Private Variable Naming

> To make code as readable as possible, try to use self explanatory variable names and avoid using single character variable names.

Reviewer: ```suggestion
To improve the readability of the code, try to use self-explanatory variable names and avoid using single-character variable names.
```

Example: `_velocity` instead of `_v`

#### 1.1.2 Global Variable Naming

All global variables must start with the ZEN prefix followed by the component, separated by underscores. Global variables may not contain the `fnc_` prefix if the value is not callable code.

Example: `zen_component_variableName`

> _For ZEN this is done automatically through the usage of the `GVAR` macro family._

Reviewer: ```suggestion
_For ZEN this is done automatically through the use of the `GVAR` macro family._
```
